mkt/developers/templates/developers/apps/status.html

```diff
@@ -373,13 +373,7 @@ <h2 id="low-memory-devices">{{ _("Low-Memory Devices") }}</h2>
               {{ _('View your most recently submitted test plan') }}</a></p>
           {% endif %}
           <p class="listing-footer">
-            {% set btn_txt = _('Learn how to get your app pre-installed on Firefox OS devices') %}
-            {% if test_plan %}
-              {% set btn_txt = _('Update your test plan') %}
-            {% endif %}
-            <a class="button" href="{{ addon.get_dev_url('preload_' + next_step_suffix) }}">
-              {{ btn_txt }}
-            </a>
+             <a href="{{ addon.get_dev_url('preload_' + next_step_suffix) }}">Click here</a> to learn how to get your app pre-installed on Firefox OS devices.
```

Comment:
You'll need to make sure the text is localised, so it can be translated. In the above examples, that's what the _('...') is for.

The previous text changes from "Learn how..." to "Update your..." depending if a a test plan already exists. Seems like a reasonable thing to do. Should we still do that?